src/shogun2-core/src/main/java/de/terrestris/shogun2/model/layer/util/WmtsTileGrid.java

```diff
@@ -0,0 +1,67 @@
+package de.terrestris.shogun2.model.layer.util;
+
+import de.terrestris.shogun2.model.layer.util.TileGrid;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * Class representing a WMTS tile grid
+ *
+ * @author Andre Henn
+ */
+@Entity
+@Table
+public class WmtsTileGrid extends TileGrid {
+
+	// String array holding the matrix IDs.
+	private String[] matrixIds;
```

Comment:
How would this be persisted?

Reply:
In order to be consistent with the other models in SHOGun2, I'll switched to `List<String>` (annotated with `@ElementCollection(targetClass = String.class)`) here.